ocp_resources/task.py

```diff
@@ -0,0 +1,77 @@
+from ocp_resources.resource import MissingRequiredArgumentError, NamespacedResource
```

Comment:
Did you use the script to generate the class?

Reply:
No I will use it. Thanks for pointing

Reply:
@myakove  maybe i missed something but i see this issue https://github.com/RedHatQE/openshift-python-wrapper/issues/2006